terraform/projects/people-depot/people-depot-dev.tf

```diff
@@ -34,9 +34,16 @@ module "people_depot_backend_stage_service" {
    container_port = 8000
    container_image = "${module.people_depot_ecr_backend.repository_url}:dev"
    container_environment = [
+      { "name": "DEBUG", "value": "false"},
       { "name": "SECRET_KEY", "value": random_password.cookie_key.result},
       { "name": "DJANGO_PORT", "value": "8000"},
       { "name": "DJANGO_ALLOWED_HOSTS", "value": module.people_depot_backend_stage_dns_entry.full_dns_name},
+      { "name": "SECURE_HSTS", "value": "31536000"},
```

Comment:
This should be `SECURE_HSTS_SECONDS` to match the [other PR](https://github.com/hackforla/peopledepot/pull/467)'s variable name.